stablehlo/dialect/TypeInference.h

```diff
@@ -268,13 +329,22 @@ LogicalResult inferTupleOp(MLIRContext* context, Optional<Location> location,
                            ValueRange val,
                            SmallVectorImpl<Type>& inferredReturnTypes);
 
+LogicalResult inferUniformDequantizeOp(
+    Optional<Location> location, Value operand,
+    SmallVectorImpl<ShapedTypeComponents>& inferredReturnShapes);
+
 LogicalResult inferWhileOp(Optional<Location> location, ValueRange operand,
                            SmallVectorImpl<Type>& inferredReturnTypes);
 
```

Comment:
What should be our plan for `reifyReturnTypeShapes`? I know that we have postponed figuring this out until shape functions are finished, but now that we're close to completion, I figured I'd bring this up.

Reply:
I do not have clear idea of this for the moment: create https://github.com/openxla/stablehlo/issues/828 to track this and will add my founds there.